extensions/wikia/Recirculation/js/helpers/sponsoredContent.js

```diff
@@ -0,0 +1,97 @@
+define('ext.wikia.recirculation.helpers.sponsoredContent', [], function () {
+	'use strict';
+
+	var userGeo = Geo.getCountryCode();
+
+	function fetch() {
+		var mock = [
+            {
+                "id": 0,
+                "url": "https://starwars.wikia.com/wiki/Yoda",
+                "thumbnailUrl": "https://vignette.wikia.nocookie.net/starwars/images/d/d6/Yoda_SWSB.png/revision/latest/scale-to-width-down/500?cb=20150206140125",
+                "weight": 20,
+                "geos": [
+                    "US"
+                ],
+                "title": "Yoda",
+                "siteName": "Hulu"
+            },
+            {
+                "id": 0,
+                "url": "https://muppet.wikia.com/wiki/Elmo",
+                "thumbnailUrl": "https://vignette.wikia.nocookie.net/starwars/images/d/d6/Yoda_SWSB.png/revision/latest/scale-to-width-down/500?cb=20150206140125",
+                "weight": 25,
+                "geos": [],
+                "title": "Elmo",
+                "siteName": "Hulu"
```

Comment:
how about `attribution` for field name here and the second one would be `attributionLabel`?